### DISCUSSION TITLE
Can we just call it NRed?.

### DISCUSSION THREAD

**Discussion by CraigHazan** [ChefKissInc/NootedRed#139] [Polls]
Over the past year we've grown to love WhateverRed, NootedRed and NRed, but it gets a little confusing with so many references. I've noticed that 'NRed' has become a popular and endearing nick name for the beloved whatever you call it and propose we leave it at that, (it's also a acronym for Nerd, which is awesome).

**Comment by VisualEhrmanntraut** [comment]
WhateverRed is the old name, which was associated with WhateverGreen, Acidanthera, Dortania and AMD OSX. The name was last used >4 months ago, before the switch to NootedRed. If you still have trouble switching your brain to call it NootedRed or NRed, nothing I can do. 
NRed is just a shortened version of NootedRed. The "Nooted" is just an emphasis on the Noot branding, that we're no Acidanthera, Dortania or AMD OSX.

**Reply by CraigHazan** [reply]
God you're so toxic some times!. It's If **you're** still having trouble switching  your brain to call it NootedRed or NRed, **there is** nothing I can do'
Did you notice in the vote above that one is having problems 'switching' there brains to NRed or NootedRed?.

**Reply by VisualEhrmanntraut** [reply]
I was half joking about the fact some people still have trouble comprehending the name change. Using the old name will cause new users to be confused and/or associate us with the aforementioned organisations. Also, there was nothing grammatically incorrect about my sentence. Additionally, I'm the least toxic developer out there.
Btw, their* brains. And !. -> !, and you're missing a ' after "It's". And a full stop after "nothing I can do'". And ?. -> ?

**Reply by CraigHazan** [reply]
Thanks, I appreciate the English lesson. should be consistent though, NRed is catching on with people.

**Reply by oq-x** [reply]
Ok NeRd

**Comment by samgreat1402** [comment]
Who give you that permission to choose name for someone else project!!

**Reply by CraigHazan** [reply]
Who gave you permission to deny me some fun!. There's a distinct lack of humor here, apart from ChefKissinc who is comedy gold!.
